lib/Doctrine/Parser/Yml.php

```diff
@@ -45,10 +47,10 @@ class Doctrine_Parser_Yml extends Doctrine_Parser
     public function dumpData($array, $path = null, $charset = null)
     {
         try {
-            $data = sfYaml::dump($array, 6);
+            $data = Yaml::dump($array, 6);
```

Comment:
Any idea why CodeClimate is complaining about this?
> Call to method `dump` from undeclared class `\Symfony\Component\Yaml\Yaml`

Reply:
I'm not really sure, it comes from Phan. I feel like it's doing it because it's a static method call perhaps?

That or it doesn't understand the `use` statement properly.